scripts/symbolize.py

```diff
@@ -366,6 +370,24 @@ def write(self, line):
                 return
             else:
                 self.reset()
+        if self._func_graph_found:
+            match = re.search(GRAPH_ADDR_RE, line)
+            match_re = re.search(GRAPH_RE, line)
+            if match:
+                addr = match.group('addr')
+                pre = match.start('addr')
+                post = match.end('addr')
+                self._out.write(line[:pre])
+                res = self.resolve(addr)
+                res_arr = re.split(' ', res)
+                self._out.write(res_arr[0])
+                self._out.write(line[post:])
+                return
+            elif match_re:
+                self._out.write(line)
+                return
+            else:
+                self.reset()
```

Comment:
This may be the cause of the problem I mentioned in another comment (dump cannot be decoded if it doesn't start as expected after the "Function graph" line). I think we should be more robust against weird input.

Reply:
Sure, will make it more robust.